frontend/views.py

```diff
@@ -65,3 +70,40 @@ def get_page(request, file_name):
 
 def redirect_images_to_media(request, image):
     return redirect(settings.MEDIA_URL + "images/" + image)
+
+def submit_page_change(request):
+
+    if request.method == 'POST':
```

Comment:
You can use this decorator to make the code cleaner https://docs.djangoproject.com/en/2.0/topics/http/decorators/#django.views.decorators.http.require_POST